engine/src/engine.ts

```diff
@@ -205,6 +208,15 @@ export default class Engine {
     // }
   }
 
+  isVersionOrLater(version: string) {
+    if (!this.version) return false;
+    return this.version >= version;
```

Comment:
Use the [semver-compare](https://www.npmjs.com/package/semver-compare) package - already added

It will process correctly comparisons between `5.10.0` and `5.9.0`